server/chat-commands/info.ts

```diff
@@ -1552,7 +1552,7 @@ export const commands: Chat.ChatCommands = {
 			`§ <strong>Section Leader</strong> - They oversee rooms in a particular section`,
 			`% <strong>Global Driver</strong> - Like Voice, and they can lock users and check for alts`,
 			`@ <strong>Global Moderator</strong> - The above, and they can globally ban users`,
-			`* <strong>Global Bot</strong> - Like Moderator, but makes it clear that this user is a bot`,
+			`* <strong>Global Bot</strong> - They can use HTML anywhere`,
```

Comment:
Should these continue to mention that the rank is intended for accounts controlled by software?